wiring/src/spark_wiring_logging.cpp

```diff
@@ -472,6 +284,12 @@ int spark::detail::LogFilter::nodeIndex(const Vector<Node> &nodes, const char *n
 
 // spark::StreamLogHandler
 void spark::StreamLogHandler::logMessage(const char *msg, LogLevel level, const char *category, const LogAttributes &attr) {
+    // TODO: Move this check to a base class (see also JSONStreamLogHandler::logMessage())
+#if PLATFORM_ID != PLATFORM_GCC
+    if (stream_ == &Serial && Network.listening()) {
```

Comment:
Will this also quiet logs sent to Serial1? I know Brett relies on Serial1 logging for deep dive into connectivity issues, even in listening mode.

Reply:
No, this change will only affect logs sent to `Serial`

Reply:
Bravo, this is a good bandaid to fixing broken listening mode!